app/controllers/claims_controller.rb

```diff
@@ -8,6 +8,7 @@ class ClaimsController < BasePublicController
   before_action :check_claim_not_in_progress, only: [:new]
   before_action :clear_claim_session, only: [:new]
   before_action :prepend_view_path_for_policy
+  before_action :persist, only: [:new]
```

Comment:
Suggestion: move the `persist` call inside the `new` action for slightly improved clarity.